chall09/egarcia-.c

```diff
@@ -0,0 +1,179 @@
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <ctype.h>
+
+char	*ft_strjoin(char const *s1, char const *s2)
+{
+	char	*str;
+	int		i;
+
+	i = 0;
+	if (s1 && s2)
+	{
+		if (!(str = (char*)malloc((strlen(s1) + strlen(s2) + 1)
+			* sizeof(char))))
+			return (NULL);
+		while (*s1)
+			str[i++] = *s1++;
+		while (*s2)
+			str[i++] = *s2++;
+		str[i] = '\0';
+		return (str);
+	}
+	return (0);
+}
+
+char   *ft_print_format(int y ,int d, int h, int m, int s)
```

Comment:
This conversion doesn't fit the required type, here you are converting a signed int. Which means that if you go over `INT_MAX` you will go around and lose the initial value.  
For example, if you convert `9999999999999999999 seconds`, you will get `317097919837` which is higher than `2,147,483,647` (`INT_MAX`).  
Thus you won't pass this test.

Type choosing is really important in programmation, it has to fit the needs!